cpp/src/parquet/bloom_filter.h

```diff
@@ -311,9 +311,12 @@ class PARQUET_EXPORT BlockSplitBloomFilter : public BloomFilter {
   ///
   /// @param properties The parquet reader properties.
   /// @param input_stream The input stream from which to construct the Bloom filter.
+  /// @param bloom_filter_length The length of the Serialized Bloom filter including
```

Comment:
```suggestion
  /// @param bloom_filter_length The length of the serialized bloom filter including
```

The capitalization of `Bloom` in the above line also looks strange to me. It would either be `Bloom Filter` or `bloom filter`.